.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: python
 
+os:
+    - linux
+    - osx
```

Comment:
Eventually it'll work, but there are tons I don't know about how travis sets up the tests, and the additional sauce I'll have to add to travis.yaml.